### ISSUE TITLE
Restore ancestral reconstruction worker

### ISSUE THREAD

**Issue by cmdcolin** [GMOD/JBrowseMSA#11]

**Comment by ihh** [comment]
I'm unclear what this issue is proposing

**Comment by cmdcolin** [comment]
this was a feature of abrowse if i was not mistaken, similar to #4

**Comment by cmdcolin** [comment]
the UI result of the feature was showing the row with 'seqlogo' style letters with modified heights, this tool doesn't have that

**Comment by ihh** [comment]
Right, gotcha. It was a nice feature but not essential I think, given that we have had to triage some things. I would for example be far more interested in putting effort into faster rendering, OpenGL, possibly even restoring animations

**Comment by cmdcolin** [comment]
Closing — restoring an in-viewer ancestral-reconstruction worker conflicts with the project's current direction: the viewer visualizes precomputed data, analysis happens externally (tutorial or CLI), the same reasoning that removed the in-app InterProScan/EBI queue submission (a2f34f9a). Reopen if this fits better as a tutorial addition instead.
